README.md

```diff
@@ -92,10 +88,10 @@ The second is less comprehensive, but can be useful for very simple commands. Th
 
 ```ruby
 result = FooCommand.call(foo: "foo")
-puts result.success? # => true
+puts result.passed? # => true
```

Comment:
The main reason for the language difference is an intersection between ruby styles and limitations from keywords.

* We used `flunk` because `fail` is a ruby keyword
* We stopped using `pass` because `pass` is a python keyword
* `ko` and `ok` are conveniently opposites of each-other, are super short, are unlikely to run into conflicts with method names you want in your command
* Stylistically (for ruby), we want these status checks to be past-tense — but `ok` and `ko` don't really have past tenses: `kod` and `koed` both look... bad, `okd` and `okayed` also look... bad. @ncallaway had a pitch to use `ok` and `err` instead of `ok` and `ko`, which at least doesn't look like `kod` — but not fully settled here.

I could be persuaded here, but I think that `result.ko?` is harder to understand at a glance than `result.failed?`, and I think that we're consistent enough: generate results with `ok`/`ko`, check results with `passed?`/`failed?`.

Reply:
I don't love `ko` but could get used to it. We could possibly do `ok?` and `not_ok?` for checks